backend/src/data/repositories/index.ts

```diff
@@ -7,6 +7,7 @@ import { Document } from './document-repository';
 import { Notification } from './notification.repository';
 import { DoctorRepository } from './doctor-repository';
 import { PermissionRepository } from './permission-repository';
+import { MessagesRepository } from './messages-repository';
```

Comment:
```suggestion
import { MessagesRepository } from './messages.repository';
```
Rename all by this way